src/main/java/core/basesyntax/vehicles/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Excavator is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Excavator is stopped");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative and indicate the type of machine, which is `Excavator` in this case.